aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -92,87 +89,58 @@ const filtersConfig = [
-                  Users
-                </Heading>
-                <Box direction="row" justify="between" align="start" wrap>
-                  <FilterControls
-                    // Table column configuration should be grouped on right
-                    // side with other actions to separate it from filtering.
-                    // If grouped to the left with the Filter control, it
-                    // becomes confusing what the "Clear filters" button will
-                    // do with regards to any column configurations that have
-                    // been applied.
-                    configure={
-                      <DropButton
-                        a11yTitle="Configure columns button"
-                        icon={<Splits />}
-                        kind="toolbar"
-                        dropAlign={{ top: 'bottom', right: 'right' }}
-                        onClose={() => setOpen(false)}
-                        onOpen={() => setOpen(true)}
-                        dropContent={
-                          <ColumnSettings
-                            columns={COLUMNS}
-                            visibleColumns={visibleColumns}
-                            setVisibleColumns={setVisibleColumns}
-                            open={open}
-                          />
-                        }
-                        tip="Configure columns"
-                      />
-                    }
-                    actions={<Menu kind="toolbar" label="Actions" items={[]} />}
-                    data={allData}
-                    filters={filtersConfig}
-                    searchFilter={{ placeholder: 'Search' }}
-                  />
-                </Box>
-              </Box>
-            </Header>
-            <Results columns={visibleColumns} />
+export const TableCustomizationExample = () => (
+  <Page background="background" fill>
+    <PageContent>
+      <Box gap="medium">
+        <Header pad={{ top: 'medium' }}>
+          <Box gap="xsmall" fill="horizontal">
+            <Heading id="users-heading" level={2} margin="none">
+              Users
+            </Heading>
           </Box>
-        </FiltersProvider>
-      </PageContent>
-    </Page>
-  );
-};
+        </Header>
+        <Results />
+      </Box>
+    </PageContent>
+  </Page>
+);
 
-const Results = ({ columns }) => {
+const Results = () => {
   const [select, setSelect] = useState([]);
-  const { filteredResults } = useFilters();
 
   return (
-    <Box fill overflow="auto">
-      <DataTable
-        aria-describedby="users-heading"
-        data={filteredResults}
-        background="background"
-        columns={columns}
-        select={select}
-        onSelect={setSelect}
-        pin
-      />
+    <Box fill>
+      <Data data={allData} updateOn="change" fill>
+        <Toolbar>
+          <Box direction="column" fill>
+            <Box direction="row" gap="small">
+              <DataSearch />
```

Comment:
<img width="1074" alt="Screen Shot 2023-01-23 at 11 44 34 AM" src="https://user-images.githubusercontent.com/12522275/214135324-14d9a082-4e71-41da-9f66-7618a91b9e8f.png">

the search input feels a little narrow, I feel like commonly I've seen it at width="medium" if there's space -- can we look at how to let the search take up more width when it's available?

also, on narrow screens the search was very narrow, we should likely set a min width as well. I'd expect the controls to wrap more like in https://storybook.grommet.io/?path=/story/layout-toolbar-simple--simple

Reply:
This was something that I brought up in the notes section, I wasn't sure if I should let the DataSearch component handle it or force width="medium" here. Will add the min width and note that down!

Reply:
DataSearch should behave as desired out of the box and not need extra customizations for the 80% use case. So, we should build this into DataSearch itself.

Reply:
Going to leave this comment here for now and keep the width="medium" until added into DataSearch component.